maps/randomvaults/objects.dm

```diff
@@ -136,3 +139,20 @@
 /obj/item/weapon/paper/iou
 	name = "paper- 'IOU'"
 	info = "I owe you a rod of destruction. Redeemable at Milliway's at the end of time."
+
+/obj/machinery/floodlight/on
+
+	New()
```

Comment:
OH GOD, I CAN EXPLAIN

SEE THE VOX CHICKEN THING, I DID THE SAME MISTAKE

"I asked in IRC, and all I got was memes"